linker/shared/src/main/scala/org/scalajs/linker/backend/emitter/Emitter.scala

```diff
@@ -997,6 +1025,15 @@ final class Emitter(config: Emitter.Config, prePrinter: Emitter.PrePrinter) {
     }
   }
 
+  // case class to get automatic equals()
+  private case class UncachedDecisions(
```

Comment:
I'm wondering if we should bit-pack this (feel we have done this for less in the past :P).